### ISSUE TITLE
HDR playback Nvidia shield

### ISSUE THREAD

**Issue by forkymcforkface** [yuliskov/SmartYouTubeTV#315]
The shield has been updated to android pie and now has the vp9.2 profile. HDR is not working on sytv though.

**Comment by bumar2000** [comment]
Yes, that is true. According to Google vp9.2 can be decoded by software in Pie. Is that possible to enable that in SYTV?

**Comment by forkymcforkface** [comment]
Yes, it looks like it just needs to be switched on. @yuliskov are you able to do this?

**Comment by fgl27** [comment]
some context see.
https://github.com/yuliskov/SmartYouTubeTV/issues/390

**Comment by fgl27** [comment]
quadrupled issue
https://github.com/yuliskov/SmartYouTubeTV/issues/242
https://github.com/yuliskov/SmartYouTubeTV/issues/315
https://github.com/yuliskov/SmartYouTubeTV/issues/390
https://github.com/yuliskov/SmartYouTubeTV/issues/573

can someone close this issue as makes no sense https://github.com/yuliskov/SmartYouTubeTV/issues/242 is enough.
and can people be polite and stop open so many issues about the same thing.
